constants/index.js

```diff
@@ -0,0 +1,10 @@
+// File for all project constant
```

Comment:
I thought this file will be created in another PR. This PR creates the constants file AND update the AppNav now.